lib/user-interface/react/src/components/model-management/ModelManagementActions.tsx

```diff
@@ -113,8 +113,8 @@ function ModelActionButton (dispatch: ThunkDispatch<any, any, Action>, notificat
         items.push({
             text: 'Update',
             id: 'editModel',
-            disabled: true,
-            disabledReason: 'This action will be available with LISA release 3.0.1',
+            disabled: !selectedModel,
```

Comment:
Non blocking: We don't technically need to do this since the parent dropdown is disabled if we don't have a model selected.

Reply:
Yeah I went back and forth on this. I liked this because if we change when the menu is disabled this would keep working. But it is superfluous now so I can remove it.